onadata/settings/docker.py

```diff
@@ -37,9 +37,9 @@
 # Make a unique unique key just for testing, and don't share it with anybody.
 SECRET_KEY = '~&nN9d`bxmJL2[$HhYE9qAk=+4P:cf3b'
 
-ALLOWED_HOSTS = ['127.0.0.1', 'localhost']
+ALLOWED_HOSTS = ['127.0.0.1', 'localhost', '0.0.0.0']
```

Comment:
Codacy found an issue: [Possible binding to all interfaces.](https://app.codacy.com/manual/Ona/onadata/pullRequest?prid=5964884)